yarn-project/types/src/abi/contract_artifact.ts

```diff
@@ -285,6 +285,10 @@ function getNoteTypes(input: NoirCompiledContract) {
         nullable: field.value.fields[1].value.value,
       };
     });
+    // TODO(benesjan): Nuke this once you nuke hacky_partial_note.nr
```

Comment:
Maybe link this to an issue? I find these standalone todos easy to forget.

Reply:
I've already nuked that in a [PR](https://github.com/AztecProtocol/aztec-packages/pull/8951) up the stack so will not get forgotten